docs/DESIGN.md

```diff
@@ -17,14 +17,14 @@ actionable buttons.
 Previously we have been displaying notifications for each archive and hence the notifications feeds were also configured to fetch data based on the archive. Our design has never
 considered providing users with an ability to take action right from the notification feed. In recent years, new features haven't been rolled out with accompanying notifications,
 which has left the notifications feed limited in the types of notices it displays to users.
-@Bryson new design has combined all feeds into one account notification feed, and also has actionable buttons corresponding to each feed. The limitations of the current
-notification system calls for a redesign and re-architecture.
+The new UI ![design](..\notification_ui_design.png) has combined all feeds into one account notification feed, and also has actionable buttons corresponding to each feed. The limitations of the current notification system calls for a redesign and re-architecture.
```

Comment:
This link isn't working for me. It looks like `DESIGN.md` and `notification_ui_design.png` are both in `docs/` - which is good! - so you shouldn't need to change directories at all: `![design](notification_ui_design.png)`.

Once that is fixed, I think you'll find that dropping the image in-line like that is going to disrupt the flow. The image is 1600x1258, which is a bit large for text to just wrap around. Can you rephrase so that the text accommodates a large illustration?